java/identify-kml-features/README.md

```diff
@@ -18,28 +18,30 @@ Note: the KML layer used in this sample contains a screen overlay. The screen ov
 
 1. Create an `OnTouchListener` on the `MapView`.
 2. On tap:
-  * Dismiss the callout.
+  * Dismiss the `Callout`, if one is showing.
   * Call `identifyLayersAsync(...)` passing in the `KmlLayer`, screen point and tolerance.
```

Comment:
Layer should be singular in the method call, and it probably makes sense to include the class name as well.
```suggestion
  * Call `MapView.identifyLayerAsync(...)` passing in the `KmlLayer`, screen point and tolerance.
```